content/en/docs/components/katib/overview.md

```diff
@@ -0,0 +1,205 @@
++++
+title = "Introduction to Katib"
+description = "Overview of Katib for hyperparameter tuning and neural architecture search"
+weight = 10
+                    
++++
+
+{{% beta-status
+  feedbacklink="https://github.com/kubeflow/katib/issues" %}}
+
+Katib is a Kubernetes-native project for automated machine learning (AutoML) —
+it's a system for hyperparameter tuning and neural architecture search.
+Katib supports a number of machine learning frameworks, including
+TensorFlow, MXNet, PyTorch, XGBoost, and others.
+Learn more about AutoML at [fast.ai](https://www.fast.ai/2018/07/16/auto-ml2/),
+[Google Cloud](https://cloud.google.com/automl),
+[Microsoft Azure](https://docs.microsoft.com/en-us/azure/machine-learning/concept-automated-ml#automl-in-azure-machine-learning) or
+[Amazon SageMaker](https://aws.amazon.com/blogs/aws/amazon-sagemaker-autopilot-fully-managed-automatic-machine-learning/).
+
+Katib supports a lot of various AutoML algorithms, such as
+Bayesian optimization, Tree of Parzen Estimators, Random Search,
+Covariance Matrix Adaptation Evolution Strategy, Hyperband,
+Neural Architecture Search based on ENAS, Differentiable Architecture Search and
+many more. Further algorithms are under development.
```

Comment:
In the entire paragraph, I'd recommend adding links to arXiv and other original posts that describe each AutoML algorithm in detail (you did that in another guide that listed the algorithms). Also, maybe it's best to spell out ENAS? Just a suggestion.

```suggestion
Katib supports a lot of various AutoML algorithms, such as
Bayesian optimization, Tree of Parzen Estimators, Random Search,
Covariance Matrix Adaptation Evolution Strategy, Hyperband,
Neural Architecture Search based on ENAS, Differentiable Architecture Search and
many more. Additional algorithm support is coming soon.
```

Reply:
Was Google's NAS with RL paper the earliest research published on AutoML, btw? 🤔 "Neural Architecture Search with Reinforcement Learning" - https://arxiv.org/abs/1611.01578

Reply:
Yes, it was one of the earliest.
@8bitmp3 Do we want to add it also ?
Our ENAS implementation is based on this paper: https://github.com/kubeflow/katib/tree/master/pkg/suggestion/v1beta1/nas/enas#about-the-efficient-neural-architecture-search.

Reply:
It was just a suggestion - thought it could be relevant if you found it some use. Up to you